test_functions.h

```diff
@@ -0,0 +1,37 @@
+#ifndef TEST_FUNCTIONS_H
+#define TEST_FUNCTIONS_H
+
+#define ASSERT_EQ(expected, actual) do { \
+if ((expected) != (actual)) { \
+        printf("TEST FAILED!!! Bad cred in ovector: expected %d, actual %d but we continue\n", (expected), (actual)); \
+        return; \
+} \
+} while(0)
+
+void runTests();
+
+void initVector_conditionsDefault_shouldCreateArray();
+
+void deleteArray__conditionsBad_shouldDeleteArray();
+
+void pushBack__conditionsGood_shouldAddtoArrayInBack();
```

Comment:
for me it's better to use less words in test_case names

for example: (try to use the following pattern)
* not this `pushBack__conditionsGood_shouldAddtoArrayInBack`
* but this `pushBack__addToArrayInBack`